merino-adm/src/remote_settings/mod.rs

```diff
@@ -43,19 +42,28 @@ impl RemoteSettingsSuggester {
         config: &RemoteSettingsConfig,
         metrics_client: StatsdClient,
     ) -> Result<Box<Self>, SetupError> {
-        let mut remote_settings_client = RemoteSettingsClient::new(
-            &settings.remote_settings.server,
-            config
-                .bucket
-                .as_ref()
-                .unwrap_or(&settings.remote_settings.default_bucket)
-                .clone(),
-            config
-                .collection
-                .as_ref()
-                .unwrap_or(&settings.remote_settings.default_collection)
-                .clone(),
-        )?;
+        let mut remote_settings_client = remote_settings_client::Client::builder()
```

Comment:
Previously we were utilizing a `FileStorage`, e.g. `remote_settings_client.storage(Box::new(FileStorage..`, do we want to do that again? cc @mythmon

Reply:
Yes, if we don't we'll be re-downloading attachments at every sync.